astroid/nodes/as_string.py

```diff
@@ -65,7 +65,7 @@ def _precedence_parens(self, node, child, is_left=True):
 
         return child.accept(self)
 
-    def _should_wrap(self, node, child, is_left):
+    def _should_wrap(self, node, child, is_left: bool) -> bool:
```

Comment:
How is `is_left` determined? The `!=` check seems weird for a bool.

Reply:
It looks like callers just specify it. It's always boolean as far as I can tell.